### ISSUE TITLE
Releases?

### ISSUE THREAD

**Issue by scopatz** [hydrocomputing/pymf6#1]
Will there be any official releases of this package?

**Comment by pya** [comment]
Well, there is:

https://anaconda.org/hydrocomputing/pymf6

**Comment by scopatz** [comment]
Right, but I would like to put this on conda-forge, which requires a version / tarball

**Comment by pya** [comment]
Great, would love to have pymf6 on conda-forge. :) Can you elaborate what steps I need  to do get this repo ready to be added to conda-forge.

**Comment by scopatz** [comment]
Basically, it just needs a version tag on github here or a release tarball uploaded to github releases
